doc/source/data/training_ingest.rst

```diff
@@ -0,0 +1,525 @@
+                train_loop_per_worker=train_func,
+                scaling_config=ScalingConfig(num_workers=2, use_gpu=use_gpu),
+                datasets={"train": train_dataset},
+            )
+            result = trainer.fit()
+
+            # Extract the model from the checkpoint.
+            result.checkpoint.to_dict()["model"]
+
+
+        .. testoutput::
+            :hide:
+
+            ...
+
+    .. group-tab:: XGBoost
+
+        Run `pip install -U xgboost-ray`
+
+        .. testcode::
+
+            import ray
+            from ray.train.xgboost import XGBoostTrainer, XGBoostCheckpoint
+            from ray.air.config import ScalingConfig
+
+            # Step 1: Create a Ray Dataset from a CSV file.
+            # You can also create a Ray Dataset from many other sources and file
+            # formats.
+            dataset = ray.data.read_csv("s3://anonymous@air-example-data/breast_cancer.csv")
+
+            # Split data into train and validation.
+            train_dataset, valid_dataset = dataset.train_test_split(test_size=0.3)
+
+            # XGBoost does not require defining your own training logic, so skip steps
+            # 2 and 3
+
+            # Step 4: Create a XGBoosTrainer. 
+            # Specify the number of training workers and pass in your Ray Dataset.
+            # The Ray Dataset is automatically split across all training workers.
+            trainer = XGBoostTrainer(
+                scaling_config=ScalingConfig(num_workers=2, use_gpu=False),
+                label_column="target",
+                num_boost_round=20,
+                params={
+                    # XGBoost specific params
+                    "objective": "binary:logistic",
+                    "eval_metric": ["logloss", "error"],
+                },
+                datasets={"train": train_dataset, "valid": valid_dataset},
+            )
+            result = trainer.fit()
+
+            # Extract the model from the checkpoint.
+            XGBoostCheckpoint.from_checkpoint(result.checkpoint).get_model()
+
+        .. testoutput::
+            :hide:
+
+            ...
+
+.. _data_train_examples:
+
+More examples
+-------------
+- :doc:`Fine-tuning PyTorch FasterRCNN_Resnet50 object detection model </ray-air/examples/torch_detection>`
+- :doc:`Distributed training with XGBoost </ray-air/examples/xgboost_example>`
+
+.. _train_datasets_configuration:
+
+Configuration and troubleshooting
+---------------------------------
+This section covers how to configure Ray Datasets for training, including performance tips.
+
+Customizing how to split datasets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By default, Ray Train will split the ``"train"`` dataset across workers using :meth:`Dataset.streaming_split <ray.data.Dataset.streaming_split>`. This means that each worker sees a disjoint subset of the data, instead of iterating over the entire dataset. For all other datasets, the entire dataset is passed to each worker.
```

Comment:
Can we discuss whether the splittings are identical across different iterations?

Reply:
do you mean if they are shuffled?

Reply:
yeah. If they are shuffled or not, and if each worker uses the same split for each iteration.